src/current/v23.2/alter-virtual-cluster.md

```diff
@@ -0,0 +1,101 @@
+
+Use the [`GRANT SYSTEM`]({% link {{ page.version.version }}/grant.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT SYSTEM MANAGEVIRTUALCLUSTER TO user;
+~~~
+
+## Synopsis
+
+<div>
+{% include {{ page.version.version }}/physical-replication/alter-virtual-cluster-diagram.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------+------------
+`virtual_cluster_spec` | The virtual cluster's name.
+`PAUSE REPLICATION` | Pause the replication stream.
+`RESUME REPLICATION` | Resume the replication stream.
+`COMPLETE REPLICATION TO` | Set the time to complete the replication. Use: <br><ul><li>`SYSTEM TIME` to specify a [timestamp]({% link {{ page.version.version }}/as-of-system-time.md %}). Refer to [Cut over to a point in time]({% link {{ page.version.version }}/cutover-replication.md %}#cut-over-to-a-point-in-time) for an example.</li><li>`LATEST` to specify the most recent replicated timestamp. Refer to [Cut over to a point in time]({% link {{ page.version.version }}/cutover-replication.md %}#cut-over-to-the-most-recent-replicated-time) for an example.</li></ul>
+`SET REPLICATION RETENTION = duration` | Change the [duration]({% link {{ page.version.version }}/interval.md %}) of the retention window that will control how far in the past you can [cut over]({% link {{ page.version.version }}/cutover-replication.md %}) to.
+`GRANT ALL CAPABILITIES` | Grant the virtual cluster all [capabilities]({% link {{ page.version.version }}/create-virtual-cluster.md %}#capabilities).
+`REVOKE ALL CAPABILITIES` | Revoke all [capabilities]({% link {{ page.version.version }}/create-virtual-cluster.md %}#capabilities) from the virtual cluster.
+`GRANT CAPABILITY virtual_cluster_capability_list` | Specify a [capability]({% link {{ page.version.version }}/create-virtual-cluster.md %}#capabilities) to grant to the virtual cluster.
+`REVOKE CAPABILITY virtual_cluster_capability_list` | Revoke a [capability]({% link {{ page.version.version }}/create-virtual-cluster.md %}#capabilities) from the virtual cluster.
+`RENAME TO virtual_cluster_spec` | Rename a virtual cluster.
+`START SERVICE SHARED` | Start the virtual cluster. That is, start the standby's virtual cluster so it is ready to accept SQL connections after cutover.
+`STOP SERVICE` | Stop the `shared` service for the virtual cluster. Note that the virtual cluster's `data_state` will remain as `ready` for the service to be started once again.
+
+## Examples
+
+### Start the cutover process
+
+To start the [cutover]({% link {{ page.version.version }}/cutover-replication.md %}) process, use `COMPLETE REPLICATION` with the necessary timestamp:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER VIRTUAL CLUSTER standbyapplication COMPLETE REPLICATION TO LATEST;
```

Comment:
Should `LATEST` here perhaps be `<CUTOVER TIME SPECIFICATION>` where then the time specification is decribed below as either `LATEST` or `SYSTEM TIME <TIME>`

Reply:
oh yeah, I like that... it makes me want to update our `RESTORE` docs in a similar way 😬